target/linux/ramips/rt305x/base-files/etc/board.d/02_network

```diff
@@ -172,6 +172,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0:lan:3" "1:lan:2" "2:lan:1" "3:lan:0" "4:wan" "6@eth0"
 		;;
+	zyxel,keenetic-4g-b)
+		ucidef_add_switch "switch0" \
+		"0:lan:1" "1:lan:0" "4:wan" "6@eth0"
```

Comment:
Ditto.
And please add one more tab in this line.

Reply:
moved.
added.